### PR TITLE
Add `torch.nn.Hardswish()` and `torch.nn.functional.hardswish()`

### PR DESCRIPTION
**Pull request by kaiidams**
- Add torch.nn.Hardswish() and torch.nn.functional.hardswish()
- `torch.nn.functional.glu(): fix type of `dim`.
- `torch.nn.functional.hardsigmoid(): Call Tensor.hardsigmoid directly.
- `torch.nn.functional.hardsigmoid(): Uncapitalize function name
- `torch.nn.functional.leaky_relu(): Wrong default value of negative_slope. https://pytorch.org/docs/stable/generated/torch.nn.functional.leaky_relu.html#torch.nn.functional.leaky_relu
- `ChannelShuffle.cs` moved as it is not an activation function.

### CODE REVIEW

File: src/TorchSharp/NN/Activation/Hardswish.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                /// Applies the Hardswish function, element-wise, as described in the paper:
+                /// `Searching for MobileNetV3 https://arxiv.org/abs/1905.02244`.
+                /// </summary>
+                /// <param name="input">The input tensor</param>
+                /// <param name="inplace">Do the operation in-place</param>
+                /// <returns></returns>
+                static public Tensor hardswish(Tensor input, bool inplace = false)
+                {
+                    using (var m = nn.Hardswish(inplace)) {
+                        return m.forward(input);
```
Comment (NiklasGustafsson) [inline src/TorchSharp/NN/Activation/Hardswish.cs:62]: Maybe this can just invoke Tensor.hardswish instead of creating a module?

**Review by NiklasGustafsson — CHANGES_REQUESTED** [review]
Use Tensor.hardswish in the nn.functional version.

File: src/TorchSharp/NN/Activation/Hardswish.cs (hunk 1 above)
Comment (kaiidams) [reply src/TorchSharp/NN/Activation/Hardswish.cs]: Thanks. Fixed `hardsigmoid()` and `hardswish()`

**Review by NiklasGustafsson — APPROVED** [review]
